numpy_challenge/numpy_challenge.py

```diff
@@ -0,0 +1,67 @@
+# Installing and importing numpy
+
+!pip install numpy
+
+import numpy as np
+
+# Creating the arrays
+
+data = [1, 2, 3]
+my_first_array = np.array(data) # The first way is making the array from existing data
+
+my_second_array = np.full(8, 8) # Creating the array from zero. We get array with 8 eights
+
+my_third_array = np.arange(1, 10, 2) # We get the array from 1 to 10 with the step 2
+
+# Creating matrix_multiplication
+
+def matrix_multiplication(m1, m2):
+    return m1 @ m2
+
+# Creating multiplication_check
+
+def multiplication_check(ms):
+    result = True
+    for i in range(len(ms)):
+        if i > 0:
+            shape_prev = ms[i-1].shape
+            shape_i = ms[i].shape
+            if shape_prev[1] != shape_i[0]:
+                result = False
+    return result
```

Comment:
Подход верный, но можно немного упростить
```suggestion
    for i in range(len(ms) - 1):
            if ms[i].shape[1] != ms[i+1].shape[0]:
                return False
    return True
```